packages/backend/src/api/controllers/ForcedTradeOfferController.ts

```diff
@@ -152,7 +167,24 @@ function validateBalance(
   return false
 }
 
-export function validateSignature(
+export function validateInitialSignature(
+  offer: Omit<ForcedTradeInitialOfferRecord, 'createdAt' | 'id'>,
+  signature: string,
+  ethAddressA: EthereumAddress
+) {
+  const stringOffer = JSON.stringify(
```

Comment:
I'd prefer being explicit here with:
```ts
JSON.stringify({
  field1: value1,
  field2: value2,
  ...
})
```
This way the field order is clear and the transformation can be done in place too.